jssource/src_files/modules/Calendar/Cal.js

```diff
@@ -1099,6 +1098,10 @@ $($.fullCalendar).ready(function () {
             },
             show: {solo: true},
             hide: {event: false},
+							event: 'mouseleave',
```

Comment:
your code is indented out of line with the rest of the code in this section, in this case I would suggest the indents should be consistent with the rest of the code in the file.